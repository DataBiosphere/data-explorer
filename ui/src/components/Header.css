.toolbar {
    background-color: #553A70 !important;
    height: 100px !important;
    margin-bottom: 15px;
}

.datasetName {
    color:white !important;
    font-size: 30px;
    font-weight: bold;
    padding-left: 30px;
}

.participantCountBox {
    color:white !important;
}

.totalCountText {
    font-size: 25px;
    text-align: center;
}

.separator {
    color: white !important;
    background-color: white !important;
    margin-left: 15px;
    margin-right: 15px;
}