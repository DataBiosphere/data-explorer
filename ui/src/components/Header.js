import React from "react";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

function Header(props) {
  const datasetName = props.datasetName;
  const totalCount = props.totalCount;

  return (
    <AppBar position="static" style={{ backgroundColor: "#5aa6da" }}>
      <Toolbar>
        <Typography className="datasetName" variant="headline" color="inherit" style={{ flexGrow: 1 }}>
          {datasetName}
        </Typography>
        <Typography className="totalCountText" color="inherit">
          {totalCount} Participants
        </Typography>
      </Toolbar>
    </AppBar>
  );
}

export default Header;
