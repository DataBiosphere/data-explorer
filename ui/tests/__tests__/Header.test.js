import React from "react";
import Header from "../../src/components/Header";

test("Renders with input", () => {
  const tree = shallow(
    <Header datasetName={"Dataset Name"} totalCount={100} />
  );
  expect(tree).toMatchSnapshot();
});
