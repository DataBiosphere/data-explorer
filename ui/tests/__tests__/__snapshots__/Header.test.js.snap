// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders with input 1`] = `
<Toolbar
  className="toolbar"
  noGutter={false}
>
  <ToolbarGroup
    firstChild={true}
    lastChild={false}
  >
    <ToolbarTitle
      className="datasetName"
      text="Dataset Name"
    />
  </ToolbarGroup>
  <ToolbarGroup
    firstChild={false}
    lastChild={false}
  >
    <div
      className="participantCountBox"
    >
      <div
        className="totalCountText"
      >
        100
      </div>
      <div>
        Participants
      </div>
    </div>
  </ToolbarGroup>
</Toolbar>
`;
